bundles/fluent-en_GB/atrium-fluent-en_GB/src/jvmTest/kotlin/custom/SmokeSpec.kt

```diff
@@ -12,25 +12,27 @@ import ch.tutteli.atrium.logic.createDescriptiveAssertion
 import ch.tutteli.atrium.reporting.Text
 import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
 import ch.tutteli.atrium.translations.DescriptionBasic.TO_BE
-import org.spekframework.spek2.Spek
+import io.kotest.core.spec.style.DescribeSpec
 import java.nio.file.Paths
 
-object SmokeSpec : Spek({
-    test("see if `toEqual` can be used") {
-        expect(1).toEqual(1)
-    }
+object SmokeSpec : DescribeSpec({
```

Comment:
kotest requires the spec to be a class, that's why it could not discover this spec
```suggestion
class SmokeSpec : DescribeSpec({
```

Reply:
Thanks!
Actually I looked into the Spek documentation and there also implementation was done with class. But in Atrium it was using object so I thought we have to stay with object in Kotest as well.

Reply:
@robstoll updated the PR kindly look into it.